packages/contracts-bedrock/src/L1/opcm/OPContractsManagerV2.sol

```diff
@@ -281,6 +294,11 @@ contract OPContractsManagerV2 is ISemver, OPContractsManagerUtilsCaller {
         view
         returns (bool)
     {
+        // The interop migration instruction should always be permitted regardless of the OPCM version.
+        if (LibString.eq(_instruction.key, Constants.INTEROP_MIGRATION_ADDRESSES)) {
```

Comment:
I think we should actually do the interop dev flag check here. It's a more reliable and legible place to do that check.